core/src/main/java/feast/core/model/Field.java

```diff
@@ -214,6 +225,40 @@ public Field(EntitySpec entitySpec) {
     }
   }
 
+  public Map<String, String> getLabels() {
+    ObjectMapper mapper = new ObjectMapper();
```

Comment:
in TypeUtils we have helper functions `convertJsonStringToMap` and `convertMapToJsonString`, you might want to use those for converting to and from json.